spec/factories/posts.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :post do
+    user
+    section
```

Comment:
I thought so too, but then I read this: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations

What do you make of that, Ken? I was a little confused. It seems to work, though!